### ISSUE TITLE
Using AI Platform Batch Prediction for Evaluator component

### ISSUE THREAD

**Issue by casassg** [tensorflow/tfx#3946]
**Describe the feature and the current behavior/state.**

Currently Evaluator uses in Beam inference in order to evaluate the model. It may be interesting to use AI Platform Batch Prediction API instead to perform inference, similar to how BatchInferrer does it.

This is a feature request, but at the same time we can contribute it back. Would appreciate pointers on how could this be developed.

**Will this change the current API? How?**

It may need extending Spec to accept configuration needed for AI Platform Prediction setup.

**Who will benefit with this feature?**

TFX users with Evaluator using it from GCP.

**Do you have a workaround or are completely blocked by this?** :

Works currently, mostly just an optimization.

**Name of your Organization (Optional)**

Twitter

**Any Other info.**

This is low priority, but would be nice to get pointers on how to implement internally if we want to.

**Comment by singhniraj08** [comment]
@casassg,

Are you still looking for a resolution? We are planning on prioritising the issues based on the community interests. Please let us know if this issue still persists with the latest TFX 1.13 release so that we can work on fixing it. 
Currently [google_could_ai platform](https://www.tensorflow.org/tfx/api_docs/python/tfx/v1/extensions/google_cloud_ai_platform) supports BulkInferrer, Pusher, Trainer and Tuner.
Thank you for your contributions.
